spec/classes/init_spec.rb

```diff
@@ -119,17 +119,24 @@
         end
 
         it do
-          is_expected.to contain_file("/opt/keycloak-#{version}/config.cli").only_with(
-            ensure: 'file',
+          is_expected.to contain_concat("/opt/keycloak-#{version}/config.cli").with(
+            ensure: 'present',
             owner: 'keycloak',
             group: 'keycloak',
             mode: '0600',
-            content: %r{.*},
             notify: 'Exec[jboss-cli.sh --file=config.cli]',
             show_diff: 'false',
           )
         end
 
+        it do
+          is_expected.to contain_concat__fragment("config.cli-keycloak").with(
```

Comment:
Use single quotes, rubocop is failing here. One way to fix if you have Ruby environment is `bundle exec rake rubocop:auto_correct`.

Reply:
Great, thanks I didn't know it. I just `push -f` the fix.